### ISSUE TITLE
Edit Mode - Optimize Mesh tools panel

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#105]
We have some inconsistencies in the Mesh tools panel that needs a cleanup and improvement. 

In Text mode we could group the buttons in groups of four buttons like in the icon mode.
There is a gap above the Merge Separate menus caused by a wrong layout separator in the above code.
In icon mode we have made the extrude dropdown box a single button. There is now space for three more buttons.
Not so often used tools could go to the bottom. Deform below Modify. Merge and Separate above Modify maybe. Or we keep the label away and put merge and separate at the top besides the two transform tools. And we could put the knife and equal tools into its own blocks. 

In short, let's toy around here a bit.

![meshtools](https://user-images.githubusercontent.com/13919702/37822675-6dd6251a-2e87-11e8-93ee-7ea521466ec0.jpg)

**Comment by ReinerBforartists** [comment]
Fixed in workflow branch

https://github.com/Bforartists/Bforartists/commit/4879597a67be4d48dcab141ba90a5c4cc93ab21a

![optimized](https://user-images.githubusercontent.com/13919702/37824926-7784f95e-2e8e-11e8-8f79-446bcf6ac62e.jpg)
